cirq/aqt/aqt_device.py

```diff
@@ -26,50 +26,32 @@
 import json
 from typing import Union, Tuple, List, Dict, Sequence, Any, cast
 import numpy as np
-from cirq import ops, devices, study
-from cirq import Circuit, LineQubit, IonDevice, Duration
-from cirq import DensityMatrixSimulator
 
-gate_dict = {'X': ops.X, 'Y': ops.Y, 'Z': ops.Z, 'MS': ops.XX}
+from cirq.ops import MeasurementGate, \
+    X, Y, Z, XX,\
+    ZPowGate, XPowGate, YPowGate, XXPowGate,\
+    Moment, Operation, Qid,\
+    depolarize, measure
```

Comment:
I think you can add parentheses around the imports to avoid the "\"s

I think the convention is to say "from cirq import ops" and then use ops.X below.  That way you don't need to list every single operation in ops.